sdk/core/azure-core/azure/core/pipeline/_base.py

```diff
@@ -105,6 +105,12 @@ def send(self, request):
         )
 
 
+def _implements_sansio_policy_protocol(obj):
```

Comment:
Should we have this in policies/_base.py to make it live with SansIOHTTPPolicy definition?

Reply:
Good idea, have done.